templates/terraform/encoders/monitoring_slo.go.erb

```diff
@@ -15,13 +15,14 @@
 // Name/Service Level Objective ID is a query parameter and cannot
 // be given in data
 delete(obj, "sloId")
-basSli := obj["serviceLevelIndicator"].(map[string]interface{})["basicSli"].(map[string]interface{})
-
-//Removing the dummy `enabled` attribute
-	if availability, ok := basSli["availability"]; ok {
+Sli := obj["serviceLevelIndicator"].(map[string]interface{})
```

Comment:
Should we check if `serviceLevelIndicator` is present too, or is it guaranteed to always be returned?

Reply:
A slo should have at least one ServiceLevelIndicator, Goal & calendar period.